src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -681,6 +707,52 @@ public final void testGetPatronAccountById() {
     logger.info("Test done");
   }
 
+  @Test
```

Comment:
Can you please add a test cases that exercise:
* limit < 0 (should get all records back)
* limit = 0 and include*=true (should return just the summary info)

Reply:
Updated